addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/HiddenFilesScanRule.java

```diff
@@ -48,10 +48,9 @@
  * <p>Initial payload set adapted from snallygaster (by Hanno Böck):
  * https://github.com/hannob/snallygaster
  *
- * <p>TODO:<br>
- * - Implement support for checking byte arrays in responses (could use HexString handling similar
- * to Replacer).<br>
- * - Implement support for "not" conditions in content checks.
+ * <p><strong>Note:</strong> Binary Regex matching assumes:<br>
```

Comment:
I'd suggest removing the "Regex", it's not a regex (it just uses similar escapes).

Reply:
Per comment in IRC, I think just "binary" is fine.